staging/src/k8s.io/client-go/pkg/util/exec/exec.go

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
```

Comment:
I noticed that module `pkg/util/parsers` was used in client-go in the same way, thats way i copied `pkg/util/exec` instead of moving. Also i wasnt sure that `pkg/util/exec` belongs to client-go